### PR TITLE
Sync master to live

### PR DESCRIPTION
**Pull request by asm-foysal**
Sync master to live according to https://github.com/Samsung/tizen-docs/pull/1668#issuecomment-1208915373

### CODE REVIEW

File: docs/application/web/api/6.5/ui_fw_api/Wearable_UIComponents/wearable_checkbox.htm [hunk 1]
```diff
@@ -16,14 +16,6 @@ <h1>Checkbox and Radio Button</h1>
...
-      onclick="
-         if (navigator.userAgent.toLowerCase().indexOf('chrome') == -1) {
-            alert('Chrome Browser required')
-         } else
-            window.open('https://code.tizen.org/demos?path=1.1%2Fexamples%2Fwearable%2FUIComponents%2Fcontents%2Fcontrols%2Fcheckbox.html', '_blank')"
-   >Use example</button>
-   </button>
 <br>
 <h2>Radio Button</h2>
 <br>
```
Comment (mijin85cho) [inline docs/application/web/api/6.5/ui_fw_api/Wearable_UIComponents/wearable_checkbox.htm:21]: I found the button in line 32~38 is not removed. 
Would you please check it with the author of the PR? If the button should be removed also, please ask the author to raise the fix PR.
If the fix PR is merged to master, this PR 1671 will automatically include the fix PR.

Same issue to 6.0.

If the author says it is OK to include the button, I'll approve PR 1671.

Comment (singa2000) [reply docs/application/web/api/6.5/ui_fw_api/Wearable_UIComponents/wearable_checkbox.htm:21]: Thanks for your checking.
I also remove the button and make PR for it by https://github.com/Samsung/tizen-docs/pull/1672

Comment (asm-foysal) [reply docs/application/web/api/6.5/ui_fw_api/Wearable_UIComponents/wearable_checkbox.htm:21]: @mijin85cho the PR [1672 ](https://github.com/Samsung/tizen-docs/pull/1672) has been merged.

**Review by safir-srbd — APPROVED** [review]
@asm-foysal @mijin85cho I am approving this PR for merging.

**Review by mijin85cho — APPROVED** [review]
Thanks for the update.

**Comment by singa2000** [comment]
@mijin85cho @asm-foysal 
Thanks for your quick checking.
When can it be reflected in https://docs.tizen.org/?

**Comment by mijin85cho** [comment]
Hello, @asm-foysal 
Since the last live-master sync, it passed more than a month. 
How about raising a PR to sync the live branch with the master branch?

**Comment by asm-foysal** [comment]
@mijin85cho hello!
We were planning on doing that after reviewing and updating all the files in the Platform section of the Tizen Docs site: https://docs.tizen.org/platform/
We finally finished our review and merged all the updates today, we'll raise a PR for syncing to live soon.
Thank you for the reminder though, we really appreciate it!

**Comment by mijin85cho** [comment]
@asm-foysal Hello.
I saw the PR https://github.com/Samsung/tizen-docs/pull/1687 was raised up. Thanks for your reply, and rapid F/U.

**Pull request merged**